### ISSUE TITLE
Xcode Error: linker command failed with exit code 1 (use -v to see invocation)

### ISSUE THREAD

**Issue by Immerence** [Unity-Technologies/arfoundation-samples#736]
**Unity bug report case number**

Related to; Case 1312735.

**Describe the bug**

Archive in Xcode; Error: linker command failed with exit code 1 (use -v to see invocation) when Archive in Xcode.

**To Reproduce**

Using 2020.2.1f1, ARKit 4.0.10, AR Foundation 4.0.10, Universal RP 10.2.2 & Xcode version 11.3.1.

- Build & export project to Xcode. 
- Wait for successful build.
- Archive project in Xcode.
- See, Error: linker command failed with exit code 1 (use -v to see invocation).

**Expected behavior**

Successful archiving.

**Actual behavior**

Error. 

According to https://github.com/Unity-Technologies/arfoundation-samples/issues/588

The issue is related to not using Xcode 12 Beta, @mdurand42 comments;

If you are using the 4.1.0-preview packages then you must build your project using Xcode 12 beta. 
If you need to use Xcode 11 to build your project you should use the "4.0.x verified packages".

In this project following packages are in use;
ARKit 4.0.10, AR Foundation 4.0.10

In other words, "4.0.x verified packages". Then Xcode Version 11.3.1 should work.

**Desktop (please complete the following information):**
 - Device: Macbook Pro
 - OS: MacOS Mojave 10.14.6
 - Unity version 2020.2.1f1
 - ARFoundation version 4.0.10
 - Xcode version 11.3.1

**Comment by tdmowrer** [comment]
Xcode 12 is the only supported version. It is no longer in beta as the issue you linked to suggests. Is there any reason you can't upgrade?

**Comment by Immerence** [comment]
Alright, upgrade it is, I can run virtual machines for added backwards compatibility. 

Any way you could include this information in future releases in the documentation under technical details > requirements, https://docs.unity3d.com/Packages/com.unity.xr.arfoundation@4.0/manual/index.html#requirements

Or somewhere else I missed this info? 

Filtering the changelog did not contain any information regarding appropriate Xcode version. Thx.

**Comment by tdmowrer** [comment]
The [changelog](https://docs.unity3d.com/Packages/com.unity.xr.arkit@4.1/changelog/CHANGELOG.html) for the ARKit XR Plugin includes the version of Xcode used to build the plugin.

**Comment by Immerence** [comment]
I guess the filter for the changelog can’t handle the word Xcode. I’ll try to scroll through it instead in future releases.
